sampul/konten-en.tex

```diff
@@ -17,26 +17,26 @@
 % Ubah kalimat-kalimat berikut dengan nama dan NRP mahasiswa
 \begin{large}
   Elon Reeve Musk \\
-  \textnormal{NRP 0123 20 4000 0001}
+  \textmd{NRP 0123 20 4000 0001}
 \end{large}
 
 \vspace{\fill}
 
 % Ubah kalimat-kalimat berikut dengan nama-nama dosen pembimbing
 \begin{large}
-  \textnormal{Advisor} \\
+  \textmd{Advisor} \\
   Nikola Tesla, S.T., M.T. \\
-  \textnormal{NIP XXXXXXXXXXXXXXX} \\
+  \textmd{NIP XXXXXXXXXXXXXXX} \\
   Wernher von Braun, S.T., M.T. \\
-  \textnormal{NIP XXXXXXXXXXXXXXX}
+  \textmd{NIP XXXXXXXXXXXXXXX}
```

Comment:
i suggest to use the same NIP as in here
https://github.com/b201lab/template-proposal-ta-its/blob/102dc01384f641cc0c81fd84bff8431b4663f0e8/lainnya/lembar-pengesahan.tex#L46-L52

Reply:
done